bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/VehicleHandler.java

```diff
@@ -0,0 +1,576 @@
+                }
+            }
+        } else if (channelUID.getIdWithoutGroup().equals("clear-cache") && command.equals(OnOffType.ON)) {
+            List<String> removals = new ArrayList<String>();
+            imageStorage.get().getKeys().forEach(entry -> {
+                if (entry.contains("_" + config.get().vin)) {
+                    removals.add(entry);
+                }
+            });
+            removals.forEach(entry -> {
+                imageStorage.get().remove(entry);
+            });
+            updateState(new ChannelUID(thing.getUID(), GROUP_IMAGE, "clear-cache"), OnOffType.OFF);
+            getImageResources();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = Optional.of(getConfigAs(VehicleConfiguration.class));
+        Bridge bridge = getBridge();
+        if (bridge != null) {
+            updateStatus(ThingStatus.UNKNOWN);
+            BridgeHandler handler = bridge.getHandler();
+            if (handler != null) {
+                accountHandler = Optional.of((AccountHandler) handler);
+                startSchedule(config.get().refreshInterval);
+                if (!config.get().vin.equals(NOT_SET)) {
+                    imageStorage = Optional.of(storageService.getStorage(BINDING_ID + "_" + config.get().vin));
+                    if (!imageStorage.get().containsKey(EXT_IMG_RES + config.get().vin)) {
+                        getImageResources();
+                    }
+                    setImageOtions();
+                }
+                updateState(new ChannelUID(thing.getUID(), GROUP_IMAGE, "clear-cache"), OnOffType.OFF);
+            } else {
+                String textKey = Constants.PREFIX + "vehicle" + Constants.STATUS_BRIDGEHANDLER_MISSING;
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED, textKey);
+            }
+        } else {
+            String textKey = Constants.PREFIX + "vehicle" + Constants.STATUS_BRIDGE_MISSING;
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, textKey);
+        }
+    }
+
+    private void startSchedule(int interval) {
+        refreshJob.ifPresentOrElse(job -> {
+            if (job.isCancelled()) {
+                refreshJob = Optional
+                        .of(scheduler.scheduleWithFixedDelay(this::getData, 0, interval, TimeUnit.MINUTES));
+            } // else - scheduler is already running!
+        }, () -> {
+            refreshJob = Optional.of(scheduler.scheduleWithFixedDelay(this::getData, 0, interval, TimeUnit.MINUTES));
+        });
+    }
+
+    @Override
+    public void dispose() {
+        refreshJob.ifPresent(job -> job.cancel(true));
+    }
+
+    public void getData() {
+        if (!accountHandler.isEmpty()) {
+            String token = accountHandler.get().getToken();
+            if (EMPTY.equals(token)) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Check Bridge Authorization");
```

Comment:
Can i18n text also be provided for this description?

Reply:
Ups, missed that one